src/components/orders/OrderSurplusDisplay/index.tsx

```diff
@@ -23,7 +23,7 @@ const Wrapper = styled.div`
 `
 
 const Surplus = styled.span`
-  color: ${({ theme }): string => theme.surplusPercentage};
+  color: ${({ theme }): string => theme.green};
```

Comment:
Instead of using a specific `surplusPercentage` instead use a clear color declaration `green` which is shared in multiple places.